docs/INSTALLATION.md

```diff
@@ -74,99 +47,41 @@ improve the CMake support.
 
 [CMake 3.1.3 or later](http://www.cmake.org/) is required to use the provided build recipe.
 
-### Prerequisite: Boost 1.59+
-
-**We are in the process of transitioning away from using Boost for most
-exercises because it is difficult to install on some platforms. You can
-follow the progress, or help, [on Github](https://github.com/exercism/cpp/issues/233).**
+## Windows
 
-The unit tests use Boost.Test, the unit testing framework included with
-[Boost](http://www.boost.org/index.html).  You may find other libraries
-in Boost useful to you as you work through the exercises.
-You will need to download and install Boost.  As of this writing Boost
-1.59+ is the required version.  You will need a compiled version of the
-boost libraries Boost.Test and Boost.DateTime, or you will need to
-download from source and build the library yourself.
+Windows users can get
+[Visual Studio Community](https://www.visualstudio.com/vs/community/),
+a free download that will give you the Visual Studio IDE and the latest version
+of the Microsoft Visual C++ compiler.
 
-If you are having difficulties installing Boost for use with exercism,
-[ask for help](https://github.com/exercism/cpp/issues).
+1. Follow [the instructions here](https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio) to setup CMake.
+1. Download the exercise using the [exercism cli](https://exercism.io/cli-walkthrough) tool.
+1. Open Visual Studio 2017 or higher (With Visual C++ tools for CMake installed as described from the link above).
+1. Open the exercise folder in VS.
+1. Run "Build->Build the solution". The tests will execute with the build, with the output going to the build window. Since the test runs during the build, VS will say it is a build failure if one of the tests fail, and the test failure messages will be in the build output window.
 
-#### Linux
+You should be able to use other integrated IDE's such as Jetbrain's Clion in a similar manner.
 
-Linux users can usually get Boost from your distribution's package manager.
-If a pre-built package is unavailable for your linux distribution, try
-[installing Boost from source](http://www.boost.org/doc/libs/release/more/getting_started/index.html).
+## Linux
 
-Alternately, there may be some other way to install Boost for your
-distribution without installing from source. For example, you can install
-Boost 1.60 for Ubuntu 16.04 LTS by using a PPA.
+Ubuntu 16.04 and later have compatible compilers in the package manager, so
+installing the necessary compiler can be done with `sudo apt-get install gcc`.
```

Comment:
Consider highlighting `sudo apt-get install gcc` by moving it to a separate block.
One's eye is drawn to the more complex and frequently unnecessary installations below simply because they take up more real estate.

Reply:
I rewrote this little section to handle the common case first, since Ubuntu 16.04 is super old, and separated this command out to it's own segment. I think it's clearer now.